src/syscall.c

```diff
@@ -18,21 +18,23 @@
  * system call: name, number
  */
 /* clang-format off */
-#define SUPPORTED_SYSCALLS       \
-    _(close,            57)      \
-    _(lseek,            62)      \
-    _(read,             63)      \
-    _(write,            64)      \
-    _(fstat,            80)      \
-    _(exit,             93)      \
-    _(gettimeofday,     169)     \
-    _(brk,              214)     \
-    _(clock_gettime,    403)     \
-    _(open,             1024)    \
-    IIF(RV32_HAS(SDL))(          \
-        _(draw_frame,   0xBEEF)  \
-        _(setup_queue,  0xC0DE)  \
-        _(submit_queue, 0xFEED), \
+#define SUPPORTED_SYSCALLS           \
+    _(close,                57)      \
+    _(lseek,                62)      \
+    _(read,                 63)      \
+    _(write,                64)      \
+    _(fstat,                80)      \
+    _(exit,                 93)      \
+    _(gettimeofday,         169)     \
+    _(brk,                  214)     \
+    _(clock_gettime,        403)     \
+    _(open,                 1024)    \
+    IIF(RV32_HAS(SDL))(              \
+        _(draw_frame,       0xBEEF)  \
+        _(setup_queue,      0xC0DE)  \
+        _(submit_queue,     0xFEED)  \
+        _(setup_audio,      0xD00D)  \
```

Comment:
We may swap the magic numbers.
The term 'dood' is an informal and playful variation of the word 'dude,' which is a slang term that originated in American English. 'Dude' is often used to refer to a person, typically a man or a friend.

The setup of audio is called "baby," and then the control of audio is called "dood," which sounds reasonable.